bugsnag-plugin-android-ndk/src/main/java/com/bugsnag/android/ndk/NativeBridge.kt

```diff
@@ -0,0 +1,164 @@
+package com.bugsnag.android.ndk
+
+import android.os.Build
+import android.util.Log
+import com.bugsnag.android.NativeInterface
+import com.bugsnag.android.StateEvent
+import com.bugsnag.android.StateEvent.*
+import java.io.File
+import java.util.Observable
+import java.util.Observer
+import java.util.UUID
+import java.util.concurrent.atomic.AtomicBoolean
+import java.util.concurrent.locks.ReentrantLock
+
+/**
+ * Observes changes in the Bugsnag environment, propagating them to the native layer
+ */
+class NativeBridge : Observer {
+
+    private val lock = ReentrantLock()
+    private val installed = AtomicBoolean(false)
+    private val reportDirectory: String = NativeInterface.getNativeReportPath()
+
+    private val is32bit: Boolean
+        get() {
+            val abis = NativeInterface.getCpuAbi()
+            return !abis.toList().any { it.contains("64") }
+        }
+
+    external fun install(
+        reportingDirectory: String,
+        autoDetectNdkCrashes: Boolean,
+        apiLevel: Int, is32bit: Boolean,
```

Comment:
style nit: should `is32bit` be on a new line?

Reply:
Yep, it should. I'll update this after Si's review.